inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -370,9 +366,9 @@ void MKLDNNROIPoolingNode::getSupportedDescriptors() {
         IE_THROW() << errorPrefix << "doesn't support output with rank: " << getOutputShapeAtPort(0).getRank();
     }
 
-    if (getInputShapeAtPort(1).getStaticDims()[1] != 5) {
-        IE_THROW() << errorPrefix << "has invalid shape on 1st input: ["
-                                  << getInputShapeAtPort(1).getStaticDims()[0] << "," << getInputShapeAtPort(1).getStaticDims()[1] << "]";
+    const auto& dims = getInputShapeAtPort(1).getDims();
+    if (dims[1] != Shape::UNDEFINED_DIM && dims[1] != 5) {
```

Comment:
according to spec can't be dynamic
```suggestion
    if (dims[1] != 5) {
```

Reply:
Corrected, thank you!